internal/manifest/list.go

```diff
@@ -65,6 +69,22 @@ type ListUpdate struct {
 	MediaType string
 }
 
+type instanceOperation int
+
+const (
+	op_ignore instanceOperation = iota + 1
+	op_update
+)
+
+// ListEdit includes the fields which a List's EditInstances() method will modify.
+type ListEdit struct {
+	ListUpdate
+	Platform            *imgspecv1.Platform
+	Schema2ListPlatform *Schema2PlatformSpec
```

Comment:
I think there should be just one field; it’s the job of this subpackage to deal with the individual formats. (Or do the two have different capabilities such that the OCI one can’t represent schema2?)